### PR TITLE
feat(@vtmn/svelte): add `VtmnSnackbar` component

### PR DESCRIPTION
**Pull request by Tlahey**
Add `VtmnSnackbar` component for svelte

How it works :

- VtmnSnackbar need to be put on the main body of the application.
- VtmnSnackbarItem are used by the VtmnSnackbar to display toasts
- A store vtmnSnackbarStore can be imported everywhere on the application, you can use the function `vtmnSnackbarStore.send({ content: 'Hello world!', })` to send toast to diplay
```svelte
vtmnSnackbarStore.send({ 
  content: 'Hello world!', 
  withCloseButton: true,
  action: {
    label: 'action',
    execute: () => {
      console.log('Trigger action');
    },
  },
});
```

image

### CODE REVIEW

File: packages/sources/svelte/src/components/overlays/VtmnSnackbar/VtmnSnackbar.svelte [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const closeHandler = () => {
+    timeoutId && clearTimeout(timeoutId);
+    show = false;
+  };
+
+  $: componentClass = cn('vtmn-snackbar', 'show', className);
+  $: {
+    if (show && timeout) {
+      timeoutId = setTimeout(() => {
+        show = false;
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnSnackbar/VtmnSnackbar.svelte:44]: clearTimeout

File: packages/sources/svelte/src/components/overlays/VtmnSnackbar/VtmnSnackbar.svelte [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+   * @type {number} timeout before the snackbar disappear
+   * Default 5000
+   */
+  export let timeout = 5000;
+
+  /**
+   * @type {boolean} display close button
+   * Default false
+   */
+  export let displayCloseButton = false;
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnSnackbar/VtmnSnackbar.svelte:26]: Rename by `withCloseButton`

File: packages/showcases/core/csf/components/overlays/snackbar.csf.js [hunk 3]
```diff
@@ -7,3 +7,28 @@ export const parameters = {
     url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2796%3A12600',
   },
 };
+
+export const argTypes = {
+  content: {
+    type: { name: 'string', required: true },
+    description: 'Text display on the snackbar',
```
Comment (Tlahey) [inline packages/showcases/core/csf/components/overlays/snackbar.csf.js:14]: `Text displayed on the snackbar`

File: packages/showcases/svelte/stories/components/overlays/VtmnSnackbar/VtmnSnackbar.stories.svelte [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  title="Components / Overlays / VtmnSnackbar"
+  component={VtmnSnackbar}
+  {argTypes}
+  {parameters}
+/>
+
+<Template let:args>
+  <VtmnButton
+    on:click={() => {
+      console.log('Display snackbar');
```
Comment (Tlahey) [inline packages/showcases/svelte/stories/components/overlays/VtmnSnackbar/VtmnSnackbar.stories.svelte:20]: Remove

File: packages/sources/svelte/src/components/overlays/VtmnSnackbar/README.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+```
+npm install @vtmn/svelte
+```
+
+Now, you can use `VtmnSnackbar` on your app.
+
+# How it works
+
+First, you need to place the `VtmnSnackbar` component preferably at your
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnSnackbar/README.md:13]: First, you need to place the `VtmnSnackbar` component on the top of your application

Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnSnackbar/README.md:13]: First, you need to place the `VtmnSnackbar` component on the top of your application

File: packages/sources/svelte/src/components/overlays/VtmnSnackbar/README.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# Quick start
+
+You need to add `@vtmn/svelte` on your `package.json`
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/overlays/VtmnSnackbar/README.md:3]: + `@vtmn/icon`

**Review by lauthieb — CHANGES_REQUESTED** [review]
Thanks @Tlahey.

Just a question, I see that we can change the actionLabel, but it is possible to interact with this button? I mean add a `onclick` trigger if we click on this action button inside the snackbar?

Thanks.

**Review by jb-thery — APPROVED** [review]
Nice job !

**Review by lauthieb — APPROVED** [review]
big up @Tlahey :)

**Pull request merged**